app/models/foreman_docker/docker.rb

```diff
@@ -5,6 +5,10 @@ class Docker < ::ComputeResource
     attr_accessible :email
 
     validates :url, :format => { :with => URI.regexp }
+    validates :email, :format => { :with => /\A(([\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~]+((\.\"[\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+(\.[\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+)*\")*\.[\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~]+)*)|(\"[\w !#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+(\.[\w !#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+)*\"))
+                                  @[a-z0-9]+((\.[a-z0-9]+)*|(\-[a-z0-9]+)*)*\z/ix },
+    :length => { :maximum => 60 },
```

Comment:
I filed http://projects.theforeman.org/issues/14641, and someone else had trouble with the 60-length limit before according to http://projects.theforeman.org/issues/9584.

If you want to keep the regex, that's fine, it does seem to validate everything including weird locals like `"let's send mail"@gmail.com.com`, but I'd at least remove the the length limitation, or increase it to be 254.